src/test/java/io/vertx/mqtt/test/client/MqttClientConnectTest.java

```diff
@@ -59,9 +59,11 @@ public void connectDisconnect(TestContext context) throws InterruptedException {
   }
 
   @Test
-  public void connectDisconnectNoOptions(TestContext context) {
+  public void connectDisconnectWithIdleOption(TestContext context) {
```

Comment:
Why did you change the `connectDisconnectNoOptions` already in place ? Please create a new test for this fix.

Reply:
The funny thing is that _connectDisconnectNoOptions_ is a full duplicate of _connectDisconnect_ so i have decided to modify it instead of removing as duplicate :)

Reply:
ah !! You are right :-) So it's ok ... see only the other request changes I did